app/src/components/ResourceInfoCard.tsx

```diff
@@ -1,11 +1,12 @@
-import { Button, Grid, Paper, Text, createStyles, em, getBreakpointValue, rem } from '@mantine/core';
+import { ActionIcon, Grid, Paper, Text, createStyles, em, getBreakpointValue, rem, Tooltip } from '@mantine/core';
 import Link from 'next/link';
 import React from 'react';
 import { ResourceInfo } from '@/util/types/fhir';
-import { ExternalLink } from 'tabler-icons-react';
+import { Edit, ExternalLink } from 'tabler-icons-react';
```

Comment:
I know ExternalLink was a previous PR icon choice, but doesn't it imply that... clicking it will link externally? Maybe we could use something like Details instead? https://tabler-icons.io/i/details

Reply:
I agree - it makes sense to update this since the user isn't being taken to an external link.

I tried out a few different icons - let me know your thoughts!

`Details`:
![Screenshot 2023-06-19 at 9 47 09 AM](https://github.com/projecttacoma/measure-repository/assets/87094479/c63bf99d-3dd1-4e67-af47-6fd4418984a8)

`DotsCircleHorizontal`:
![Screenshot 2023-06-19 at 9 50 13 AM](https://github.com/projecttacoma/measure-repository/assets/87094479/26c0eef7-d996-4744-9dc1-9ddc189d7674)

`ChevronRight`:
![Screenshot 2023-06-19 at 9 52 22 AM](https://github.com/projecttacoma/measure-repository/assets/87094479/ac1fa411-1028-4727-a749-46a76fcbe999)

`SquareArrowRight`:
![Screenshot 2023-06-19 at 9 56 43 AM](https://github.com/projecttacoma/measure-repository/assets/87094479/71d85317-9795-4143-8901-918df1f7a301)

`ArrowBarRight`:
![Screenshot 2023-06-19 at 9 57 36 AM](https://github.com/projecttacoma/measure-repository/assets/87094479/bd38dd60-9871-4012-94c4-8d78fe2fe9a6)

I am leaning toward using an icon that includes a right arrow since we are routing to a new page, though it's not an external link. I'm not totally against using the `Details` icon but it is not intuitive to me just from looking at it that it routes to a new page

Reply:
I also like `SquareArrowRight` along with `DotsCircleHorizontal`